gridstatus/tests/test_nyiso.py

```diff
@@ -35,11 +35,36 @@ def test_range_two_days_across_month(self):
         last_day_of_prev_month = first_day_of_month - pd.Timedelta(days=1)
         df = self.iso.get_fuel_mix(start=last_day_of_prev_month, end=first_day_of_month)
 
-        assert df["Time"].max() >= first_day_of_month
```

Comment:
What was the reason to use greater/less than comparators for these checks instead of equals? Happy to keep these the same, I just thought equals was more precise.

Reply:
I guess the updated test may fail on the first day of the month if the data is unavailable.

Reply:
i believe it it is because a given day's file will have midnight of the next day. take a look at any complete day here: http://mis.nyiso.com/public/P-63list.htm

<img width="507" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/68cda518-b399-4790-a594-59dfcdd68ea6">